File: src/main/java/de/tomjuri/armageddon/util/Timer.java
package de.tomjuri.armageddon.util;

public class Timer {
    private long endTime = System.currentTimeMillis();

    public void start(long millis) {
        endTime = System.currentTimeMillis() + millis;
    }

    public boolean isDone() {
        return System.currentTimeMillis() >= endTime;
    }
}

File: src/main/kotlin/de/tomjuri/gemdigger/util/Timer.kt
package de.tomjuri.gemdigger.util

class Timer {

    private var endTime: Long = System.currentTimeMillis()

    fun startTimer(millis: Long) {
        endTime = System.currentTimeMillis() + millis
    }
    fun isDone(): Boolean {
        return System.currentTimeMillis() >= endTime
    }
}